hedwiki/score/HED_score_0.0.1.mediawiki

```diff
@@ -508,23 +508,23 @@ Reference:
             *** High-range <nowiki>[High ( more than 70 microV)]</nowiki>
             **** PDR-amplitude-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
 
-        ** PDR-Amplitude-asymmetry <nowiki>[A difference in amplitude between the homologous area on opposite sides of the head that consistently exceeds 50 percent. When symmetrical could be labeled with base schema 'Symmetrical' tag]</nowiki>
+        ** PDR-Amplitude-asymmetry <nowiki>[A difference in amplitude between the homologous area on opposite sides of the head that consistently exceeds 50 percent. When symmetrical could be labeled with base schema Symmetrical tag]</nowiki>
             *** PDR-amplitude-asymmetry-LowerLeft <nowiki>[Amplitude lower on the left side.]</nowiki>
             *** PDR-amplitude-asymmetry-LowerRight <nowiki>[Amplitude lower on the right side.]</nowiki>
             *** PDR-amplitude-asymmetry-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
 
-        ** PDR-frequency-asymmetry <nowiki>[When symmetrical could be labeled with base schema 'Symmetrical' tag]</nowiki>
+        ** PDR-frequency-asymmetry <nowiki>[When symmetrical could be labeled with base schema Symmetrical tag]</nowiki>
             *** PDR-frequency-asymmetry-LowerLeft <nowiki>[Hz lower on the left side.]</nowiki>
             *** PDR-frequency-asymmetry-LowerRight <nowiki>[Hz lower on the right side.]</nowiki>
 
         ** PDR-Eye-opening-reactivity <nowiki>[Change (disappearance or measurable decrease in amplitude) of a posterior dominant rhythm following eye-opening. Eye closure has the opposite effect.]</nowiki>
-            *** PDR-Eye-opening-reactivity-exists <nowiki>[When annotated 'Yes']</nowiki>
+            *** PDR-Eye-opening-reactivity-exists <nowiki>[When annotated Yes]</nowiki>
             *** PDR-Eye-opening-reactivity-reducedLeft <nowiki>[Reduced left side reactivity]</nowiki>
             *** PDR-Eye-opening-reactivity-reducedRigth <nowiki>[Reduced right side reactivity]</nowiki>
```

Comment:
Suggest changing `PDR-Eye`... to `PDR-eye` in this section.

Also change `reducedLeft` and `reducedRigth`  to `reduced-left` and `reduced-right`.